crates/jose/src/jwk/private_parameters.rs

```diff
@@ -271,7 +271,7 @@ mod rsa_impls {
         type Error = rsa::errors::Error;
         fn try_from(value: &RsaPrivateParameters) -> Result<Self, Self::Error> {
             let key: RsaPrivateKey = value.try_into()?;
-            Ok(Self::new(key))
+            Ok(Self::new_with_salt_len(key, <H as Digest>::output_size()))
```

Comment:
That was the main issue with PSS signatures. For RS* signatures, it might just be the difference between RSA 0.7.0 RC0 and RC1.